src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1390,7 +1394,7 @@ static bool processOnChangeConditionForUpdateContext
   {
     ContextAttribute* caP = notifyCerP->contextElement.contextAttributeVector[ix];
 
-    if (attrL.size() == 0)
+    if (attrL.size() == 0 || blacklist)
```

Comment:
Style-guide: parenthesis around comparison:

    if ((attrL.size() == 0) || blacklist)

[ In my opinion it's better like this:

    if ((attrL.size() == 0) || (blacklist == true))

but that's not in our style guide ]